From 41d782d5213308d591aa0e1545bf684f02fae79a Mon Sep 17 00:00:00 2001
From: ran shalit <rshalit@habana.ai>
Date: Tue, 20 Apr 2021 10:01:51 +0300
Subject: [PATCH] [SW-39246] gaudi: increase ppboot size

As preparation for next patches which adds spi
erase capability, it is required to increment
ppboot size to accommodate the additional feature.

Change-Id: I45d71828c52c5f4c98ef33800bb001dbddb26975
Signed-off-by: ran shalit <rshalit@habana.ai>
---

diff --git a/gaudi/Makefile b/gaudi/Makefile
index bc21aee..e276aa4 100644
--- a/gaudi/Makefile
+++ b/gaudi/Makefile
@@ -1,5 +1,5 @@
 FW_SIZE := 0x4000
-PPBOOT_MAX_SIZE := 16384
+PPBOOT_MAX_SIZE := 20480
 
 PREBOOT_SRCS_C += $(shell ls $(HL_DEV)/pll.c)
 PREBOOT_SRCS_C += $(shell ls $(HL_DEV)/security.c)
